apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -391,6 +391,9 @@ export default function GlobalStyle () {
                 line-height: 32px; 
                 font-weight: 700;
               }
+              .ant-modal-footer button + .ant-btn:not(.ant-dropdown-trigger) {
```

Comment:
is it possible to move it to some component?

Reply:
It's already a component, but that works with antd components only. So if we'll decide to rewrite modal form to a new uikit - we'll able to rid of GlobalStyle overrides